core/sawtooth/cli/transaction.py

```diff
@@ -155,26 +155,12 @@ def print_trans_info(args, web_client, tsctids):
         "Identifier": "TRANS",
         "InBlock": "BLOCK",
         "Status": "STATUS",
-        "TransactionType": "TXNTYPE",
-        "Update": {
```

Comment:
The way I understand this module to work is it will print transaction info for a client. Since transaction families are diverse there are only a few general fields that make sense. For now, we expect that the validator will always install the endpoint registry. Hence, these update fields are supported in this CLI helper. 
We shouldn't cut them out though when adding a new family. If you see some error condition we should figure out how to resolve that, otherwise we should drop these changes from the commit.